### PR TITLE
Updated GETSET doc #824

### LINKED ISSUE #89: Add support for QREF
QREF are Reference Queues that hold the references to other KV pairs.

While removing from the queue, we get the dereferenced entity (instead of just the key) and hence can be leveraged to make dereferenced get faster and better.

**at this stage we diverge from Redis, but for the better.

https://dicedb.arpitbhayani.me/qref

### PR DESCRIPTION
**Pull request by sashpawar11**
This PR resolves https://github.com/DiceDB/dice/issues/824

### Changes Made
- Refactored headings to match as per syntax requirement in issue https://github.com/DiceDB/dice/issues/824
- Refactored formatting for `Parameters, Return values  , Examples` sections
- Added `127.0.0.1:7379>` in all CLI implementation examples
- Refactored cli output `error` messages to reflect current CLI error message syntaxes in `Examples` and `Errors` sections
- Refactored `Behavior` section to include additional step of TTL reset on keys
- Refactored `Examples` section to include a new example displaying TTL reset on keys post `GETSET`

### CODE REVIEW

**Comment by sashpawar11** [comment]
Hi @arpitbbhayani - please review the changes.
Thankyou

**Comment by sashpawar11** [comment]
Hey @lucifercr07 @apoorvyadav1111 @AshwinKul28 - review whenever feasible.
Thankyou.

File: docs/src/content/docs/commands/GETSET.md [hunk 1]
```diff
@@ -13,44 +13,49 @@ GETSET key value
 
 ## Parameters
 
-- `key`: The key whose value you want to retrieve and set. This must be a string.
-- `value`: The new value to set for the specified key. This must be a string.
+| Parameter       | Description                                      | Type    | Required |
+|-----------------|--------------------------------------------------|---------|----------|
+| `key`           | The key whose value you want to retrieve and set.                   | String  | Yes      |
```
Comment (lucifercr07) [inline docs/src/content/docs/commands/GETSET.md:18]: Aren't these not aligned?

Comment (sashpawar11) [reply docs/src/content/docs/commands/GETSET.md:18]: My IDE had word-wrap toggled so i guess it added those spaces, but the render was accurate when previewed in the IDE. Attaching the preview shown on git below:
Screenshot From 2024-10-01 23-58-24

Let me know if we still need to refactor it.

**Review by lucifercr07 — APPROVED** [review]

**Pull request merged**